packages/strapi-plugin-content-type-builder/admin/src/translations/en.json

```diff
@@ -91,7 +94,8 @@
   "form.attribute.media.option.single.description": "Best for avatar, profile picture or cover",
   "form.attribute.media.option.single": "Single media",
   "form.attribute.settings.default": "Default value",
-  "form.attribute.text.option.long-text.description": "Best for descriptions, biography.  Exact search is disabled.",
+  "form.attribute.text.option.long-text.description": "Best for descriptions, biography. 
+Exact search is disabled.",
```

Comment:
This should be on the same line because it seems to break the build.

Reply:
Done :)